### ISSUE TITLE
Summary

### ISSUE THREAD

**Issue by gdedi001** [gbrod94/prj-rev-bwfs-dasmoto#1]
Great job with your project  in terms of the design, you met all the requirements and your project’s folder structure is very well organized. In addition to this, you used semantic tags like `<strong>` in a meaningful way. An area for improvement here is with your usage of the ​`id` and `class`​​ keywords. The way you used them leads me to believe that you are a bit confused of their purpose. 
 
A recommendation here is to read this article and apply the concepts to your project. Keep up the good work! 
[id vs. class](https://css-tricks.com/the-difference-between-id-and-class/)
[Some Best Practices](https://www.smashingmagazine.com/2008/11/12-principles-for-keeping-your-code-clean/)

**Comment by gbrod94** [comment]
Hello, thank you for the review, just to clarify to see if I am understanding what I did wrong. Since I used 3 class all with diferent values, I should actually substitute them for id?

**Comment by gdedi001** [comment]
Yup! Since they don't repeat, they are considered "unique" situations - use `id` for those.
